src/cmd/makebb/makebb.go

```diff
@@ -43,7 +43,14 @@ func main() {
 		l.Printf("Disabling CGO for u-root...")
 		env.CgoEnabled = false
 	}
+
+	err = env.CompilerInit()
+	if err != nil {
+		l.Fatal(err)
+	}
+
 	l.Printf("Build environment: %s", env)
+	l.Printf("Compiler: %s", env.Compiler.VersionOutput)
```

Comment:
This looks weird, probably one of my suggestions fits better?

```suggestion
	l.Printf("Build environment: %s\n", env)
	l.Printf("Compiler: %s\n", env.Compiler.VersionOutput)
```

Or
```suggestion
	l.Printf("Build environment: %s Compiler: %s\n", env, env.Compiler.VersionOutput)
```